Docs/source/usage/parameters.rst

```diff
@@ -2093,6 +2093,15 @@ Additional parameters
      If ``sort_intervals`` is activated and ``sort_particles_for_deposition`` is ``false``, particles are sorted in bins of ``sort_bin_size`` cells.
      In 2D, only the first two elements are read.
 
+* ``warpx.do_shared_mem_charge_deposition`` (`bool`) optional (default `false`)
```

Comment:
We haven't done as rigorous testing with charge dep as we have on current dep. We've also made some changes to Current dep that we haven't carried over to charge dep yet. Plan to mimic the work on Current Dep in Charge Dep in another PR, but since Current Dep works well, wanted to get that merged first